sqlite-android/src/main/jni/sqlite/Android.mk

```diff
@@ -26,7 +26,13 @@ sqlite_flags := \
 	-DSQLITE_ENABLE_RTREE=1 \
 	-DSQLITE_OMIT_BUILTIN_TEST \
 	-DSQLITE_OMIT_COMPILEOPTION_DIAGS \
-	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600
+	-DSQLITE_DEFAULT_FILE_PERMISSIONS=0600 \
+	-DSQLITE_DQS=0 \
+    -DSQLITE_DEFAULT_MEMSTATUS=0 \
+    -DSQLITE_MAX_EXPR_DEPTH=0 \
+    -DSQLITE_USE_ALLOCA \
+    -DSQLITE_ENABLE_BATCH_ATOMIC_WRITE \
+    -O3
```

Comment:
Seems like it must be in the next line, near LOCAL_CFLAGS.
It's classical compile optimization, like -O2, -O3, -Os